modules/microshift-multus-intro.adoc

```diff
@@ -6,21 +6,49 @@
+[id="microshift-supported-additional-networks_{context}"]
+== Supported additional networks for network isolation
+The following additional networks are supported in {microshift-short} {product-version}:
+
+* Bridge
+* IPVLAN
+* MACVLAN
+
+[NOTE]
+====
+Setting network policies for additional networks is not supported.
+====
+
+[id="microshift-additional-network-use-cases_{context}"]
 == Use case: Additional networks for network isolation
 
-You can use an additional network in situations where network isolation is needed, including control plane and data plane separation. You can create additional interfaces for pods to connect to that network in addition to a default. For example, you can configure an additional interface if you want pods to access a network on the host and also communicate with devices deployed to the edge that are on an isolated operator network or are periodically disconnected.
+You can use an additional network in situations where network isolation is needed, including control plane and data plane separation. You can create additional interfaces for pods to connect to that network in addition to a default. For example, you can configure an additional interface if you want pods to access a network on the host and also communicate with devices deployed to the edge. These edge devices might be on an isolated operator network or are periodically disconnected.
 
 Isolating network traffic is useful for the following performance and security reasons:
 
 Performance:: You can send traffic on two different planes to manage the amount of traffic on each plane.
 Security:: You can send sensitive traffic onto a network plane that is managed specifically for security considerations, and you can separate private data that must not be shared between tenants or customers.
 
-[id="additional-network-how-implemented"]
+[NOTE]
+====
+The {microshift-short} service deploys the Multus CNI plugin at start up, so a host restart might be required depending on when the `microshift-multus` RPM package is installed.
+====
+
+[id="microshift-additional-network-how-implemented_{context}"]
 == How additional networks are implemented
 
-All of the pods in the cluster still use the cluster-wide default network to maintain connectivity across the cluster. Every pod has an `eth0` interface that is attached to the cluster-wide pod network. You can view the interfaces for a pod by using the `oc exec -it <pod_name> \-- ip a` command. If you add additional network interfaces that use Multus CNI, they are named `net1`, `net2`, ..., `netN`.
+All of the pods in the cluster still use the cluster-wide default network to maintain connectivity across the cluster. Every pod has an `eth0` interface that is attached to the cluster-wide pod network. You can view the interfaces for a pod by using the `oc exec -it <pod_name> \-- ip a` command. If you add additional network interfaces that use the {microshift-short} Multus CNI, they are named `net1`, `net2`, ..., `netN`.
```

Comment:
I think we might want to mention that an annotation `k8s.v1.cni.cncf.io/network-status` exists and it contains the information but it's "the kubernetes way" and more programmatic.

https://github.com/openshift/enhancements/blob/master/enhancements/microshift/multus-cni-for-microshift.md#api-extensions

Reply:
I didn't understood this comment fully, PTAL at revision.